testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveRequireQueryPartitionsFilter.java

```diff
@@ -68,6 +68,17 @@ public void testRequiresQueryPartitionFilterOnSpecificSchema(String queryPartiti
         assertThat(onTrino().executeQuery(format("SELECT COUNT(*) FROM %s WHERE p_regionkey = 1", tableName))).containsOnly(row(5));
     }
 
+    @Test(dataProvider = "queryPartitionFilterRequiredSchemasDataProvider")
+    public void testIgnoreRequiresQueryPartitionFilterOnSpecificSchemaWhenQueryPartitionFilterRequiredIsFalse(String queryPartitionFilterRequiredSchemas)
```

Comment:
Please add an integration test instead. https://trino.io/docs/current/develop/tests.html#avoid-product-tests

Also, the test name is too long. Please change to "testSkipQueryPartitionFilterRequiredSchemas" or something.

Reply:
Thank you for detailed guide :)

I added an integration test instead, following your comment,